database/seeds/testing/13_ethnicity.seed.js

```diff
@@ -0,0 +1,17 @@
+exports.seed = (knex) =>
```

Comment:
This file is going to conflict with my pr due to file numbering. Not bothered which goes first.

Reply:
I think you'll need to rename your seed files because your user records will need valid foreign key references.